packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -2908,6 +2909,9 @@ curl -v -X POST \
 }
 ```
 
+#### Enroll Custom HOTP Factor
+Enrollment via the AuthN API is currently not supported for Custom HOTP Factor (factor type `token:hotp`).  Please refer to the Factors API documentation [here](/docs/reference/api/factors/#enroll-custom-hotp-factor) if you would like to enroll users for this type of factor.
```

Comment:
AuthN API -> Authentication API

Also I don't think we need the parenthetical information about type.